lib/lndclient/LndClient.ts

```diff
@@ -205,6 +221,14 @@ class LndClient extends BaseClient {
     return this.unaryCall<lndrpc.SendRequest, lndrpc.SendResponse.AsObject>('sendPaymentSync', request);
   }
 
+  /**
+   * Send a payment through the Lightning Network.
+   * @param lndrpc.SendRequest (see lndrpc.proto)
+   */
+  public sendPaymentSync = (request: lndrpc.SendRequest): Promise<lndrpc.SendResponse> => {
```

Comment:
I know the underlying lnd api call is `sendPaymentSync` but I think for our internal purposes it makes more sense just to call is `sendPayment` since this is technically an asynchronous method that returns only a promise.

Reply:
There is a different LND api called `sendPayment`. We better not create confusion here by calling this method `sendPayment` while it is using `sendPaymentSync`.

I wouldn't spend the time to give it another name.